source/MaterialXGenMdl/mdl/materialx/stdlib.mdl

```diff
@@ -32,6 +33,30 @@ import ::anno::*;
 // NOTE: We use the ::anno::unused() annotation to mark unsupported parameters
 
 
+// Shader Nodes
+
+export material mx_surface_unlit(
+    float mxp_emission = 0.0,
+    color mxp_emission_color = color(1.0, 1.0, 1.0),
+    float mxp_transmission = 0.0,
+    color mxp_transmission_color = color(1.0, 1.0, 1.0),
+    float mxp_opacity = 1.0
+)
+= let {
+    float trans = math::clamp(mxp_transmission, 0.0, 1.0);
```

Comment:
Small comment: You seem to perform clamps here and for osl but not for glsl.

Reply:
Since there is no explicit `1.0 - transmission` in this case I didn't add the clamp. But you are right, to be 100% consistent for values outside [0,1] I could add it here as well. Or I could remove the clamp everywhere, assuming the value is always in [0,1], which is should be by design.

Reply:
Whatever is the convention is fine. Just happen to notice this small thing.